### PR TITLE
MEX-68: Fork primaryCare into a new component

### PR DESCRIPTION
**Pull request by brandones**
https://tickets.pih-emr.org/browse/MEX-68

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -284,6 +284,10 @@ private void loadAppsAndExtensions() {
             enablePrimaryCare();
         }
 
+        if (config.isComponentEnabled(Components.MEXICO_CLINIC)) {
```
Comment (mogoodrich) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java]: @brandones currently components generally are grouped by functional areas... would be nice if we could turn this into a functional component (or use an existing one--outpatient consult?) and then potentially configure that component based on country.  Though if you think will just make it more confusing, I could possibly be persuaded this just having a Mexico Clinic is a better way... :)

**Comment by mogoodrich** [comment]
I'm fine to merge this if it's ready to go.

**Comment by brandones** [comment]
Ping, any news about this?

**Comment by mogoodrich** [comment]
Merged

**Pull request merged**